### PR TITLE
[cloudlog] new guide

### LINKED ISSUE #1285: [cloudlog] Planned new guide
Cloudlog (GitHub: https://github.com/magicbug/Cloudlog/ Website: https://www.magicbug.co.uk/cloudlog/) is a self-hosted PHP application that allows logging of amateur radio contacts anywhere.

I have just installed Cloudlog on my uberspace and plan to eventually write a guide for installation. Installation is straightforward with one problem, which I could not quickly diagnose, but easily work around.

### PR DESCRIPTION
**Pull request by SalocinHB**
closes #1285

### CODE REVIEW

File: source/guide_cloudlog.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+.. include:: includes/my-print-defaults.rst
+
+In the web form, you can leave the ``Directory`` field blank if you plan on hosting your Cloudlog instance at the root folder (i. e. ``https://isabell.uber.space/``) of your domain. The ``Website URL`` should already be pre-filled with your site's URL. You can change the ``Default Gridsquare`` if you like.
+
+For the ``Database settings``, use the hostname ``localhost``, your Uberspace user name as ``Username`` and your MySQL password as ``password``. Enter the ``Database name`` you created in the previous step (in our example this would be ``isabell_cloudlog``). Once all the required information has been entered, click install. You should now be redirected to your Cloudlog's home page. Log in with the default username ``m0abc`` and default password ``demo``, then open Admin -> User Accounts and create a new user. In order to delete the default user, log out and then log in again with your newly created Cloudlog user, head to Admin -> User Accounts again and delete ``m0abc``.
```
Comment (noave) [inline source/guide_cloudlog.rst:87]: did you check if its possible to do all this via terminal and in config files? Its a bit hard to follow. If its neccessary to do in web maybe a numbered list would be more clearly

**Review by noave — CHANGES_REQUESTED** [review]

File: source/guide_cloudlog.rst (hunk 1 above)
Comment (SalocinHB) [reply source/guide_cloudlog.rst:87]: I'm aware of it being a bit confusing, but couldn't find any CLI setup method for this in the [[.](https://github.com/magicbug/Cloudlog/wiki/Installation#5-run-the-cloudlog-install-wizard)](official documentation). A numbered list sounds like a good idea though.
